tutor/commands/config.py

```diff
@@ -185,6 +189,7 @@ def save(
     # Reload configuration, without version checking
     config = tutor_config.load_full(context.root)
     env.save(context.root, config)
+    tutor_hooks.Actions.CONFIG_LOADED.do(deepcopy(config))
```

Comment:
Can you move this action inside `load_full`? The goal is to enable anyone to check the configuration whenver the configuration is loaded, not just when we run `tutor config save`.

Reply:
Sure.